user-interface/src/main/resources/application.properties

```
@@ -186,3 +186,7 @@ spring.messages.basename=messages.error-messages, messages.toast-notifications,
 # you always have to escape single quotes by another single quote ' -> ''
 # https://www.mscharhag.com/java/resource-bundle-single-quote-escaping for reference
 spring.messages.always-use-message-format=true
+#################### Orcid API calls #########################
+qbic.orcid.api.client.id=${ORCID_API_CLIENT_ID:}
+qbic.orcid.api.client.secret=${ORCID_API_CLIENT_SECRET:}
+qbic.orcid.api.endpoint.token=${ORCID_TOKEN_URI:}
```

```suggestion
#################### Orcid API calls #########################
qbic.orcid.api.client.id=${spring.security.oauth2.client.registration.orcid.client-id}
qbic.orcid.api.client.secret=${spring.security.oauth2.client.registration.orcid.client-secret}
qbic.orcid.api.endpoint.token=${spring.security.oauth2.client.provider.orcid.token-uri}
```
No need for new properties

---

You already fixed it thanks 👍